tracing/tracing.go

```diff
@@ -173,20 +173,37 @@ func TracerFromContext(ctx context.Context) trace.Tracer {
 func makeJaegerExporter() (*jaeger.Exporter, error) {
 	var endpointOption jaeger.EndpointOption
 
+	protocol := os.Getenv("OTEL_EXPORTER_JAEGER_PROTOCOL")
+	if protocol == "" {
+		protocol = "udp/thift.compact"
+	}
+
 	// Otel Jaeger client doesn't seem to implement the spec for
 	// OTEL_EXPORTER_JAEGER_PROTOCOL selection.  So we must.
-	switch os.Getenv("OTEL_EXPORTER_JAEGER_PROTOCOL") {
+	switch protocol {
 	case "http/thrift.binary":
 		log.WithFields(logrus.Fields{
 			"endpoint": os.Getenv("OTEL_EXPORTER_JAEGER_ENDPOINT"),
-		}).Info("Initializing Jaeger exporter via http/thrift.binary")
+		}).Infof("Initializing Jaeger exporter via protocol %s", protocol)
 		endpointOption = jaeger.WithCollectorEndpoint()
 
-	default:
+	case "udp/thift.binary":
 		log.WithFields(logrus.Fields{
 			"agentHost": os.Getenv("OTEL_EXPORTER_JAEGER_AGENT_HOST"),
 			"agentPort": os.Getenv("OTEL_EXPORTER_JAEGER_AGENT_PORT"),
-		}).Info("Initializing Jaeger exporter via udp/thrift.binary")
+		}).Infof("Initializing Jaeger exporter via protocol %s", protocol)
+		endpointOption = jaeger.WithAgentEndpoint()
+
+	case "udp/thift.compact":
+		log.WithFields(logrus.Fields{
+			"agentHost": os.Getenv("OTEL_EXPORTER_JAEGER_AGENT_HOST"),
```

Comment:
want to check and or log these vars for debugging purposes?
OTEL_EXPORTER_JAEGER_AGENT_HOST
OTEL_EXPORTER_JAEGER_AGENT_PORT

Reply:
Yes, I need to know at startup to confirm where traces are going.